_component_design/buttons.md

```diff
@@ -150,18 +116,19 @@ Compact buttons should be used when you have severe space constraints, such as i
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Secondary buttons
+### Composition
 
-- The secondary button can be used alone or in conjuction with a primary button or a tertiary button.
-- Use secondary buttons when there are multiple buttons on the same screen.
-- Simple buttons, drop-down buttons, and split drop-down buttons all share the same properties for state changes.
+A simple button contains the following elements:
+
+- **Container (required)**: The container holds the call-to-action text.
+- **Call-to-action (required)**: The call-to-action text describes the act initiated when the user clicks the button.
 
 {% endcolumn %}
 
 {% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% figure [caption:"Button spacing specs"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-secondary.svg"/>
+{% figure [caption:"Simple button composition"] [class:"image bg-light border"] %}
```

Comment:
need to fix tertiary button internal spacing and add redline about how it doesn't have internal spacing.
Verify new image & redline with ryan.